numpy/core/src/umath/ufunc_object.c

```diff
@@ -2429,72 +2609,41 @@ PyUFunc_GeneralizedFunction(PyUFuncObject *ufunc,
         }
     }
     /*
-     * If keepdims is set and true, signal all dimensions will be the same.
+     * If keepdims is set and true, which means all input dimensions are
```

Comment:
Either "which" -> "this", or better, replace the period with a comma (and lower-case "signal")

Reply:
changed